src/utils/bindtwostepcarettoattribute.js

```diff
@@ -496,44 +530,33 @@ function isAtBoundary( position, attribute ) {
 // @param {module:engine/model/position~Position} position
 // @param {String} attribute
 function isAtStartBoundary( position, attribute ) {
-	const prevNode = position.nodeBefore;
-	const nextNode = position.nodeAfter;
-	const isAttrInNext = nextNode ? nextNode.hasAttribute( attribute ) : false;
-	const isAttrInPrev = prevNode ? prevNode.hasAttribute( attribute ) : false;
-
-	if ( ( !isAttrInPrev && isAttrInNext ) || isBetweenDifferentValues( position, attribute ) ) {
-		return true;
-	}
+	const { nodeBefore, nodeAfter } = position;
+	const isAttrBefore = nodeAfter ? nodeAfter.hasAttribute( attribute ) : false;
```

Comment:
I find those names confusing now. `isAttrBefore` checks `nodeAfter`. What does `isAttrBefore` mean? If anything, I'd suggest `isBeforeAttr`.

Reply:
My bad. I changed it automatically and somehow make a mistake.